README.md

```diff
@@ -18,8 +20,8 @@ To get a local copy up and running follow these simple example steps.
 ## Author
 👤 Donald Akite
```

Comment:
[ OPTIONAL ] Please make sure that your getting started section is more descriptive and the content should be the steps to follow to get your project up and running on a local machine. 


For example,
  ```
  You can add the steps to follow to clone your project,
  
  Step 1:if you don't have git installed on your machine you can download it from [here](https://git-scm.com/downloads).
  
  Once you have git installed on your machine you can clone your project by running the command below to clone your project to your local machine
  
  
  `git clone <your project link>`
  
  Alternatively, you can download the zip file of your project by clicking on the `Code` button on the right side of your project page and clicking on `Download ZIP`
  
  Step 2: Locate the folder where you cloned your project to and open the `index.html` file in your browser to view your project.
  ```